2022-04-12 16:48:36,834 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-04-12 16:48:36,834 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/settings
2022-04-12 16:48:36,835 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-12 16:48:36,835 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-12 16:48:36,835 INFO    MainThread:42619 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220412_164836-15xwub3k/logs/debug.log
2022-04-12 16:48:36,835 INFO    MainThread:42619 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220412_164836-15xwub3k/logs/debug-internal.log
2022-04-12 16:48:36,835 INFO    MainThread:42619 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9648176310>
2022-04-12 16:48:36,836 INFO    MainThread:42619 [wandb_init.py:init():439] calling init triggers
2022-04-12 16:48:36,836 INFO    MainThread:42619 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-12 16:48:36,836 INFO    MainThread:42619 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3296a9xd
2022-04-12 16:48:36,839 INFO    MainThread:42619 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/3296a9xd
2022-04-12 16:48:36,839 INFO    MainThread:42619 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-12 16:48:36,847 INFO    MainThread:42619 [wandb_run.py:_restore():1674] restore
2022-04-12 16:48:36,898 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 24241
  total_bytes: 24241
}

2022-04-12 16:48:37,232 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 24241
  total_bytes: 24241
}

2022-04-12 16:48:38,163 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 24241
  total_bytes: 29856
}

2022-04-12 16:48:38,267 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 24241
  total_bytes: 29856
}

2022-04-12 16:48:38,369 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 24241
  total_bytes: 29856
}

2022-04-12 16:48:38,471 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:38,574 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:38,676 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:38,778 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:38,880 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:38,982 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:39,838 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}

2022-04-12 16:48:40,332 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  media_count: 10
}
pusher_stats {
  uploaded_bytes: 29856
  total_bytes: 29856
}
local_info {
}

2022-04-12 16:48:41,486 INFO    MainThread:42619 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-12 16:48:41,487 INFO    MainThread:42619 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-12 16:48:41,491 INFO    MainThread:42619 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-04-12 16:48:41,498 INFO    MainThread:42619 [wandb_init.py:init():492] starting backend
2022-04-12 16:48:41,499 INFO    MainThread:42619 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-12 16:48:41,508 INFO    MainThread:42619 [backend.py:ensure_launched():219] starting backend process...
2022-04-12 16:48:41,559 INFO    MainThread:42619 [backend.py:ensure_launched():224] started backend process with pid: 50853
2022-04-12 16:48:41,561 INFO    MainThread:42619 [wandb_init.py:init():501] backend started and connected
2022-04-12 16:48:41,600 INFO    MainThread:42619 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-12 16:48:41,600 INFO    MainThread:42619 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-12 16:48:41,600 INFO    MainThread:42619 [wandb_init.py:init():565] updated telemetry
2022-04-12 16:48:41,701 INFO    MainThread:42619 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-12 16:48:43,228 INFO    MainThread:42619 [wandb_run.py:_on_init():1759] communicating current version
2022-04-12 16:48:43,295 INFO    MainThread:42619 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-12 16:48:43,296 INFO    MainThread:42619 [wandb_init.py:init():625] starting run threads in backend
2022-04-12 16:48:45,149 INFO    MainThread:42619 [wandb_run.py:_console_start():1733] atexit reg
2022-04-12 16:48:45,150 INFO    MainThread:42619 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-12 16:48:45,150 INFO    MainThread:42619 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-12 16:48:45,151 INFO    MainThread:42619 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-12 16:48:45,152 INFO    MainThread:42619 [wandb_init.py:init():664] run started, returning control to user process
2022-04-12 16:48:45,519 INFO    MainThread:42619 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-12 16:48:45,519 INFO    MainThread:42619 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-12 16:49:06,590 INFO    MainThread:42619 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-12 16:49:06,592 INFO    MainThread:42619 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-12 16:49:06,592 INFO    MainThread:42619 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-12 16:49:25,336 INFO    MainThread:42619 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-12 16:49:25,340 INFO    MainThread:42619 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-12 16:49:25,341 INFO    MainThread:42619 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-12 16:49:27,301 INFO    MainThread:42619 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-12 16:49:27,304 INFO    MainThread:42619 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-12 16:49:27,304 INFO    MainThread:42619 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-12 16:49:29,790 INFO    MainThread:42619 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/settings
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220412_164929-1mddxdov/logs/debug.log
2022-04-12 16:49:29,802 INFO    MainThread:42619 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220412_164929-1mddxdov/logs/debug-internal.log
2022-04-12 16:49:29,803 INFO    MainThread:42619 [wandb_init.py:init():439] calling init triggers
2022-04-12 16:49:29,803 INFO    MainThread:42619 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-12 16:49:29,803 INFO    MainThread:42619 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 15xwub3k
2022-04-12 16:49:29,805 INFO    MainThread:42619 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/15xwub3k
2022-04-12 16:49:29,806 INFO    MainThread:42619 [jupyter.py:save_history():429] not saving jupyter history
2022-04-12 16:49:29,806 INFO    MainThread:42619 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-12 16:49:29,806 INFO    MainThread:42619 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-04-12 16:49:29,806 INFO    MainThread:42619 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-12 16:49:29,812 INFO    MainThread:42619 [wandb_run.py:_restore():1674] restore
2022-04-12 16:49:30,373 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 913
  total_bytes: 913
}

2022-04-12 16:49:30,778 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 913
  total_bytes: 913
}

2022-04-12 16:49:31,270 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 913
  total_bytes: 5562
}

2022-04-12 16:49:31,373 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 913
  total_bytes: 5562
}

2022-04-12 16:49:31,475 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 913
  total_bytes: 5562
}

2022-04-12 16:49:31,577 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1398
  total_bytes: 5562
}

2022-04-12 16:49:31,679 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:31,782 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:31,883 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:31,985 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:32,087 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:32,189 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:32,458 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}

2022-04-12 16:49:32,969 INFO    MainThread:42619 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5562
  total_bytes: 5562
}
local_info {
}

2022-04-12 16:49:34,182 INFO    MainThread:42619 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-12 16:49:34,182 INFO    MainThread:42619 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-12 16:49:34,187 INFO    MainThread:42619 [wandb_run.py:_footer_sync_info():2822] logging synced files
